src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,23 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
```

Comment:
Implementing Cloneable and providing a custom clone method is not a common practice for entities in Hibernate. Entities are usually not cloned. Consider removing the Cloneable interface and the clone method unless there is a specific requirement for it.